PeerioTechnologies/peerio-desktop
# style: small branding fixes

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/5985/desktop-several-small-styling-fixes

#### Testing instructions  
These are all visual fixes (CSS and image files) so functionality has not been touched.

Changes:
* zero chats:
  * illustrations have been swapped
  * "plus" button icon next to "create rooms" and "create direct messages" should now be blue
* zero files
  * illustration has been swapped
* chat list
  * "Maintenance Warning" at the top should now have a red icon

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
